_data/changelogs/component-collection.yml

```diff
@@ -2,6 +2,13 @@ title: Collection
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed deprecated xlink references.
+    summaryAdditional: All deprecated xlink:href references have been replaced with href.
+    affectsMarkup: true
+    githubPr: 6165
+    githubRepo: uswds
+    versionUswds: 3.9.0
```

Comment:
**issue:** This is the wrong USWDS version number. We should update to `N.N.N` or `3.11.0` now that we've confirmed that's the next release

Reply:
Resolved in [93d5546](https://github.com/uswds/uswds-site/pull/2951/commits/93d554676a65798a6458efa3e93e7b5441596278)